cypress/e2e/specialFeatures/atiAnalytics/assertions/liteSiteCta.js

```diff
@@ -1,49 +1,30 @@
-import { liteEnabledServices } from '#app/components/LiteSiteCta/liteSiteConfig';
 import { COMPONENTS, interceptATIAnalyticsBeacons } from '../helpers';
 import { assertATIComponentClickEvent } from '.';
 
-const { LITE_SITE_CTA } = COMPONENTS;
-
-const runIfLiteEnabled = service => {
-  let run = it;
-  let skipReason = '';
-
-  if (!liteEnabledServices.includes(service)) {
-    run = it.skip;
-    skipReason = `- skipped because ${service} does not have lite enabled`;
-  }
-
-  return { run, skipReason };
-};
+const { LITE_SITE_SUMMARY } = COMPONENTS;
 
 // eslint-disable-next-line import/prefer-default-export
 export const assertLiteSiteCTAComponentClick = ({
```

Comment:
Can we rename this function to `assertLiteSiteSummaryComponentClick`

Reply:
Should we be more explicit with the naming in this particular test? As there can be 2 links in the summary, something like `assertLiteSiteSummaryComponentToMainSiteClick`?

Reply:
Even better - yes! Only that first link has tracking on it.

Reply:
Done